e2e/utils/functions/testBase.ts

```diff
@@ -308,7 +308,7 @@ export class HomePage {
     await delay(5000);
   }
 
-  async discontinueLabOrder() {
+  async discontinueAnOpenMRSEncounter() {
```

Comment:
Doesn't seem logical to discontinue an Encounter. Did you mean to void an Encounter instead? If not, then makes sense to leave the function name as is (`discontinueLabOrder`).

Reply:
> Did you mean to void an Encounter instead? 
Sure, it makes sense to rename the function to `voidEncounter()`

> stay consistent in your approach to naming methods.
:+1: